service/worker/scheduler/workflow_test.go

```diff
@@ -212,9 +212,10 @@ func (s *workflowSuite) setupMocksForWorkflows(runs []workflowRun, state *runAcr
 					s.Failf("multiple starts", "for %s at %s (prev %s)", req.Request.WorkflowId, s.now(), prev)
 				}
 				state.started[req.Request.WorkflowId] = s.now()
+				overhead := time.Duration(100+rand.Intn(100)) * time.Millisecond
```

Comment:
non-blocking comment: will the use of a random duration affect reproducibility of the tests?

Reply:
It did at first, until I reduced it to always < 1s :laughing:  It could be constant, of course, I just thought this would be a slightly better simulation